src/pyfmi/simulation/assimulo_interface_fmi2.pyx

```diff
@@ -268,59 +268,59 @@ cdef class FMIODE2(cExplicit_Problem):
         """
         The jacobian function for an ODE problem.
         """
+        _log_requires_closing_tag = False
```

Comment:
Since this is just a variable inside a function I dont see why it has to be named with leading `_`, no one will be able to access this anyway? But maybe I am missing something fundamental.